requirements-test.txt

```diff
@@ -10,7 +10,7 @@ mongobox
 ophyd
 pathlib
 pyqt5
-pytest >=3.9
+pytest >=4.4
```

Comment:
Just curious---which new feature are we employing that requires the version bump?

Reply:
pytest-rerunfailures 8.0 has requirement pytest>=4.4, but you'll have pytest 4.3.1 which is incompatible.